.gitlab/kitchen_testing/debian.yml

```diff
@@ -12,107 +12,146 @@
 .kitchen_os_debian:
   variables:
     KITCHEN_PLATFORM: "debian"
-    KITCHEN_OSVERS: "debian-8,debian-9,debian-10,debian-11"
-    DEFAULT_KITCHEN_OSVERS: "debian-11"
   before_script:
     - rsync -azr --delete ./ $SRC_PATH
     - cd $DD_AGENT_TESTING_DIR
     - bash -l tasks/kitchen_setup.sh
 
-# Kitchen: scenarios (os * agent)
+# Kitchen: scenarios (os * agent * arch)
 # -------------------------------
 
-.kitchen_scenario_debian_a6:
+.kitchen_scenario_debian_a6_x64:
+  variables:
+    KITCHEN_OSVERS: "debian-8,debian-9,debian-10,debian-11"
+    DEFAULT_KITCHEN_OSVERS: "debian-11"
+  extends:
+    - .kitchen_agent_a6
+    - .kitchen_os_debian
+    - .kitchen_azure_location_south_central_us
+  needs: ["deploy_deb_testing-a6_x64"]
+
+.kitchen_scenario_debian_a7_x64:
+  variables:
+    KITCHEN_OSVERS: "debian-8,debian-9,debian-10,debian-11"
+    DEFAULT_KITCHEN_OSVERS: "debian-11"
+  extends:
+    - .kitchen_agent_a7
+    - .kitchen_os_debian
+    - .kitchen_azure_location_south_central_us
+  needs: ["deploy_deb_testing-a7_x64"]
+
+.kitchen_scenario_debian_a6_arm64:
+  variables:
+    KITCHEN_OSVERS: "debian-10"
+    DEFAULT_KITCHEN_OSVERS: "debian-10"
   extends:
     - .kitchen_agent_a6
     - .kitchen_os_debian
-  needs: ["deploy_deb_testing-a6"]
+    - .kitchen_ec2_arm64
+  needs: ["deploy_deb_testing-a6_arm64"]
 
-.kitchen_scenario_debian_a7:
+.kitchen_scenario_debian_a7_arm64:
+  variables:
+    KITCHEN_OSVERS: "debian-10"
+    DEFAULT_KITCHEN_OSVERS: "debian-10"
   extends:
     - .kitchen_agent_a7
     - .kitchen_os_debian
-  needs: ["deploy_deb_testing-a7"]
+    - .kitchen_ec2_arm64
+  needs: ["deploy_deb_testing-a7_arm64"]
 
-# Kitchen: final test matrix (tests * scenarios)
+# Kitchen: final test matrix (tests * scenarios * arch)
 # ----------------------------------------------
 
-kitchen_debian_install_script_agent-a6:
+kitchen_debian_install_script_agent-a6_x64:
   extends:
-    - .kitchen_scenario_debian_a6
+    - .kitchen_scenario_debian_a6_x64
     - .kitchen_test_install_script_agent
 
-kitchen_debian_install_script_agent-a7:
+kitchen_debian_install_script_agent-a6_arm64:
+  extends:
+    - .kitchen_scenario_debian_a6_arm64
+    - .kitchen_test_install_script_agent
+
+kitchen_debian_install_script_agent-a7_x64:
   # Run install script test on branches, on a reduced number of platforms
   rules:
     !reference [.on_default_kitchen_tests_a7]
   extends:
-    - .kitchen_scenario_debian_a7
+    - .kitchen_scenario_debian_a7_x64
+    - .kitchen_test_install_script_agent
+
+kitchen_debian_install_script_agent-a7_arm64:
+  # Run install script test on branches, on a reduced number of platforms
```

Comment:
Comment doesn't match what the test does (it doesn't run on branches), I think it can be removed.